tests/test_delay.ts

```diff
@@ -206,7 +206,7 @@ describe('Delayed jobs', function () {
     await worker.close();
   });
 
-  it('should process delayed jobs with several workers respecting delay', async function () {
+  it.skip('should process delayed jobs with several workers respecting delay', async function () {
```

Comment:
Now instead of skipping the test when running bullmq test I changed it to skip here and we will just run yarn test